public/views/preferencesDeleteWallet.html

```diff
@@ -6,14 +6,26 @@
 
 
 <div class="content preferences" ng-controller="preferencesDeleteWalletController as preferences">
+  <div class="onGoingProcess" ng-show="isDeletingWallet">
+    <div class="onGoingProcess-content" ng-style="{'background-color':index.backgroundColor}">
+    <div class="spinner">
+      <div class="rect1"></div>
+      <div class="rect2"></div>
+      <div class="rect3"></div>
+      <div class="rect4"></div>
+      <div class="rect5"></div>
+    </div>
+    <span translate>Deleting Wallet...</span>
+    </div>
+  </div>
 
   <div class="text-center text-gray m20t size-12">
     <div class="text-warning size-18 m10b" translate>Warning!</div>
     <div class="m20b" translate>Permanently delete this wallet. THIS ACTION CANNOT BE REVERSED</div>
   </div>
   <ul class="no-bullet m0">
-    <li ng-click="preferences.deleteWallet()">
-      <div class="right" ng-style="{'color':index.backgroundColor}">
+    <li ng-click="isDeletingWallet || preferences.deleteWallet()">
```

Comment:
The function `deleteWallet()` should set the flag `isDeletingWallet = true`, not *ng-click*.